src/main/resources/db/changelog/changes/v1.0.0/sql/2021-07-21-03__create-field-configuration-table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE field_configuration
```

Comment:
it would be good to have a common prefix for these tables if they are not designed be used for anything but MARC settings. 
I.e. **marc_**: `marc_field_configuration`, `marc_ignored_prefixes` etc. This way they are grouped together and it's easier to understand their purpose.